packages/mui-base/src/BadgeUnstyled/BadgeUnstyled.js

```diff
@@ -1,35 +1,24 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import { unstable_capitalize as capitalize } from '@mui/utils';
 import composeClasses from '../composeClasses';
 import appendOwnerState from '../utils/appendOwnerState';
 import useBadge from './useBadge';
 import { getBadgeUtilityClass } from './badgeUnstyledClasses';
 
 const useUtilityClasses = (ownerState) => {
-  const { variant, anchorOrigin, invisible, classes } = ownerState;
+  const { invisible, classes } = ownerState;
```

Comment:
You are no longer propagating the `classes` prop, it was removed from props and the `ownerState`

Reply:
Great catch!